SignalServiceKit/src/Messages/DeviceSyncing/OWSSyncContactsMessage.m

```diff
@@ -87,12 +87,24 @@ - (NSData *)buildPlainTextAttachmentDataWithTransaction:(YapDatabaseReadWriteTra
             [self.identityManager recipientIdentityForRecipientId:signalAccount.recipientId];
         NSData *_Nullable profileKeyData = [self.profileManager profileKeyDataForRecipientId:signalAccount.recipientId];
 
-        TSContactThread *contactThread = [TSContactThread getOrCreateThreadWithContactId:signalAccount.recipientId transaction:transaction];
+
+        OWSDisappearingMessagesConfiguration *_Nullable disappearingMessagesConfiguration;
+        NSString *conversationColorName;
+        
+        TSContactThread *_Nullable contactThread = [TSContactThread getThreadWithContactId:signalAccount.recipientId transaction:transaction];
+        if (contactThread) {
```

Comment:
We now use a ReadTransaction which means contactThread can be nil. This is the primary change.